nmdc_server/crud.py

```diff
@@ -475,10 +475,15 @@ def get_zip_download(db: Session, id: UUID) -> Optional[str]:
             logger.warning(f"Data object url for {file.path} was {data_object.url}")
             continue
 
+        if not data_object.file_size_bytes:
+            logger.warning(
+                f"Data object file_size_bytes for {file.path} was {data_object.file_sizey_bytes}"
```

Comment:
I think there was a typo here:

```suggestion
                f"Data object file_size_bytes for {file.path} was {data_object.file_size_bytes}"
```